### PR TITLE
Add Angular elements and web component build

### PR DESCRIPTION
**Pull request by amberramesh**
- Add scripts for packing/concatenating web component
- Add module for creating wc
- Add service to append wc to app root

### CODE REVIEW

File: src/app/app.component.html [hunk 1]
```diff
@@ -1,3 +1 @@
-<div id="app">
-  <tissue-blocks></tissue-blocks>
-</div>
+<div id="app"></div>
```
Comment (bherr2) [inline src/app/app.component.html]: Why can't you use the cns tissue block web component here? Also you don't need id="app" (bad practice in Angular)

File: src/app/app.component.ts [hunk 2]
```diff
@@ -1,15 +1,17 @@
...
   styleUrls: ['./app.component.css']
 })
 export class AppComponent {
-  title = 'tissue-bar-graphs'
+  title = 'tissue-blocks'
 
-  public constructor(private titleService: Title) {
-    this.titleService.setTitle('Tissue Bar Graphs')
+  public constructor(private titleService: Title, private tissueBlockService: TissueBlocksService) {
+    this.titleService.setTitle('CNS Tissue Blocks')
```
Comment (bherr2) [inline src/app/app.component.ts]: Remove this and just set the title in the index.html

File: src/app/app.component.ts [hunk 3]
```diff
@@ -1,15 +1,17 @@
...
+import { TissueBlocksService } from './services/tissue-blocks.service';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.css']
 })
 export class AppComponent {
-  title = 'tissue-bar-graphs'
+  title = 'tissue-blocks'
```
Comment (bherr2) [inline src/app/app.component.ts]: Remove this and just set the title in the index.html

File: src/app/app.component.ts [hunk 4]
```diff
@@ -1,15 +1,17 @@
...
 
-  public constructor(private titleService: Title) {
-    this.titleService.setTitle('Tissue Bar Graphs')
+  public constructor(private titleService: Title, private tissueBlockService: TissueBlocksService) {
+    this.titleService.setTitle('CNS Tissue Blocks')
+    this.tissueBlockService.injectElement()
```
Comment (bherr2) [inline src/app/app.component.ts]: You shouldn't need to do this. You should be able to just use the custom element in your app.component.html

File: src/app/services/tissue-blocks.service.ts [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import { Injectable } from '@angular/core'
+import { NgElement, WithProperties } from '@angular/elements'
+import { TissueBlocksComponent } from '../components/tissue-blocks.component'
+
+@Injectable()
+export class TissueBlocksService {
```
Comment (bherr2) [inline src/app/services/tissue-blocks.service.ts]: Don't do this.

**Review by bherr2 — CHANGES_REQUESTED** [review]
Also, you need to update/add instructions for embedding the component to the README.md

File: src/app/app.component.html (hunk 1 above)
Comment (amberramesh) [reply src/app/app.component.html]: Updated to using web component directly (along with related changes).

File: src/app/app.component.ts (hunk 4 above)
Comment (amberramesh) [reply src/app/app.component.ts]: Have set the document title instead of using service.

**Comment by amberramesh** [comment]
>

**Review by bherr2 — APPROVED** [review]

**Pull request merged**